### PR TITLE
feat(qa): add real-path TTFV measurement

### PR DESCRIPTION
**Pull request by ebarti**
## What changed
- Updated the real-path TTFV wrapper so the gateable default path includes discovery: `jobhunter run discover score tailor --limit 1 --workers 1`.
- Removed the preselected `--job-url` path from the gate. The wrapper now rejects `--job-url`, arbitrary `--work-command`, CI, skipped phases, probe-only records, custom probe URLs, threshold overrides, and timing-only evidence.
- Added machine-checkable discovery proof: all-state pre-work `/v1/jobs` baseline hashes, selected job `discoveredAt >= T0`, hashed real discovery-source provenance, selected-row `/jobs` UI proof, and same-job Apply Review PDF byte-stream proof.
- Hardened timeout/failure records so probe timeouts keep the last observed failure and stack-health failures are written into the measurement artifact.
- Added no-spend script tests for the summary gate and wired them into `pnpm test` via `pnpm scripts:test`.
- Fixed materials validation gaps surfaced by fake-profile + real-discovery validation: education evidence is claimable, section-level education claims bind to shipped education evidence, redundant `non_requirement_reason` is canonicalized away for coverage-bearing claims, index-based generated-claim locations bind correctly, mandatory overflow accepts and audits index-based locations, and fixed education in the assembled resume counts as evidence support.
- Updated the owner protocol, QA matrix, active R2 plan amendment, local validation docs, and packaging-decision artifact to make real-path-only TTFV explicit. Synthetic/sample/fake data is not accepted for measurement.

## Constraints
- The real owner baseline was not run in this PR. It requires owner vendor auth, real discovery, real LLM spend, and owner presence.
- Fake-profile + real-discovery dev validation was run as non-gate evidence only. It proved post-T0 real discovery and scoring/UI visibility, then exposed several materials validation blockers. After the fixes, the final live fake-profile retries still did not reach Apply/PDF because the post-generation fit gate honestly rejected the fake profile against the real discovered jobs. That is not measurement evidence and is not a TTFV gate result.
- Measurement records store timings, hashes, counts, coarse host fields, timestamps, status codes, content type, and artifact byte length only. They do not store job titles, job URLs, local paths, resumes, credentials, provider logs, or profile data.
- Goal C remains a pending owner decision. The ADR records required measured inputs and options; it does not choose go/defer/no-go.

## Validation
- `node --check scripts/ttfv-real.mjs && node --check scripts/ttfv-real.test.mjs`: passed.
- `corepack pnpm scripts:test`: passed, 13/13.
- CI refusal, `--job-url` refusal, and `--work-command` refusal: exited non-zero before writing artifacts.
- Probe timeout against a closed local port: failed non-gateably and preserved the last observed failure.
- Stack-health timeout with a harmless `sleep` stack command: failed non-gateably and recorded the health timeout in `errors` and pending probes.
- `git diff --check`: passed.
- `corepack pnpm docs:build`: passed; 4812 internal references across 236 emitted files.
- `corepack pnpm check`: passed.
- `corepack pnpm test`: passed, including script tests, API tests, web build, extension unit/e2e, and Python tests; Python phase 2096 passed with one third-party deprecation warning and expected Temporal retry warnings in connection-refused test paths.
- `corepack pnpm --filter @jobhunter/web test`: passed, 210 files / 1181 tests.
- `uv --project workers/automation run --extra dev pytest -q`: passed, 2096 tests with one third-party deprecation warning and expected Temporal retry warnings in connection-refused test paths.
- `uv --project workers/automation run --extra dev ruff check .`: passed.
- PR reviewer re-check: Gate PASS, 0 findings.
- QA re-check: Gate PASS, 0 findings.

### CODE REVIEW

File: scripts/ttfv-real.mjs [hunk 1]
```diff
@@ -0,0 +1,676 @@
...
+    return { command: "", recordCommand: "skipped real job command" };
+  }
+  if (options.workCommand) {
+    const commandText = String(options.workCommand);
+    return { command: commandText, recordCommand: redactCommand(commandText) };
+  }
+  if (!options.jobUrl) {
+    throw new Error("run requires --job-url, --work-command, or --skip-work.");
+  }
+  const commandText = `uv --project workers/automation run jobhunter job ${shellQuote(String(options.jobUrl))}`;
```
Comment (ebarti) [inline scripts/ttfv-real.mjs]: PR Reviewer - Codex

Severity: High
Gate: Fail
Evidence: scripts/ttfv-real.mjs:534 builds `jobhunter job <url>` without a stop-before-apply flag; workers/automation/src/jobhunter/cli.py:1420-1423 documents that the default `jobhunter job` path runs both tailoring and apply; workers/automation/src/jobhunter/workflow_specs.py:193-196 appends the `apply` stage when `do_apply` remains true; docs/developer/first-run-ttfv.md:68-70 tells the owner to run this default wrapper.
Reproduction: Reasoned from code

The real TTFV gate is defined as first scored job and first reviewable tailored resume PDF, but the default wrapper continues into the live apply stage. That pulls tier-3 apply readiness/browser automation into an owner TTFV run, can add post-TTFV time before the record is written, and can exercise submission-adjacent behavior that is outside this measurement scope.

Required fix: Make the default work command stop after the real scoring/tailoring/PDF artifact path and before `apply` (for example an explicit tailor/materials-only command or stage set), update the docs/package examples accordingly, and require an explicit custom owner command for any apply-path experiment outside the TTFV default.

File: scripts/ttfv-real.mjs [hunk 2]
```diff
@@ -0,0 +1,676 @@
...
+function summarizeRecords(options) {
+  const files = options._;
+  if (!files.length) {
+    throw new Error("summarize requires at least one measurement record path.");
+  }
+  const thresholdTtfv1Ms = numberOption(options.thresholdTtfv1Ms, DEFAULT_TTFV_1_THRESHOLD_MS);
+  const thresholdTtfv2Ms = numberOption(options.thresholdTtfv2Ms, DEFAULT_TTFV_2_THRESHOLD_MS);
+  const worstMultiplier = numberOption(options.worstMultiplier, DEFAULT_WORST_MULTIPLIER);
+  const records = files.map((file) => JSON.parse(fs.readFileSync(file, "utf8")));
+  const successful = records.filter((record) => record.status === "passed");
```
Comment (ebarti) [inline scripts/ttfv-real.mjs]: PR Reviewer - Codex

Severity: High
Gate: Fail
Evidence: scripts/ttfv-real.mjs:190 treats every record with `status === "passed"` as summary-eligible; scripts/ttfv-real.mjs:158-166 writes probe-only records with `mode: "probe"` and a non-install T0; scripts/ttfv-real.mjs:107-126 can also record skipped install/init/stack/work phases; docs/developer/first-run-ttfv.md:50-53 defines the gate as three clean runs, and lines 91-93 say probe-only records are not clean-environment TTFV measurements.
Reproduction: Reasoned from code

`summarize` can report `status: "passed"` from three successful probe-only records, or from run records that skipped the clean-run phases, because it only filters on record status and duration fields. That makes the packaging evidence gate accept incomplete evidence even though the phase contract requires three full clean owner runs from install T0 through the real path.

Required fix: Validate each summary input before counting it: require `mode === "run"`, an install-based T0, required non-skipped phases (`install`, `workspace_init`, `stack_start`, `real_job_pipeline`), real-path policy fields, and successful TTFV-1/TTFV-2 probes. Reject probe-only/skipped-phase records with a clear error and add a focused regression test or fixture for this negative path.

File: scripts/ttfv-real.mjs [hunk 3]
```diff
@@ -0,0 +1,676 @@
...
+  } catch (error) {
+    record.probes[name].lastError = error instanceof Error ? error.message : String(error);
+  }
+}
+
+async function probeTtfv1(page, { apiBaseUrl, webBaseUrl }) {
+  const jobs = await requestJson(apiBaseUrl, "/v1/jobs?page=1&pageSize=100&sort=fit_score&dir=desc");
+  const items = Array.isArray(jobs.items) ? jobs.items : [];
+  const scored = items.filter((item) => Number.isFinite(item.fitScore));
+  const candidate = scored[0];
```
Comment (ebarti) [inline scripts/ttfv-real.mjs]: PR Reviewer - Codex

Severity: High
Gate: Fail
Evidence: scripts/ttfv-real.mjs:301-304 queries `/v1/jobs` and selects the first scored job, not the job URL passed to `run`; scripts/ttfv-real.mjs:333-347 selects the first review-queue item with a PDF and falls back to the first rendered `open final file` link; apps/web/src/views/apply-review/ApplyReviewView.tsx:80-81 defaults the UI selection to the first queue item when no `jobKey` is supplied.
Reproduction: Reasoned from code

The probes are not correlated to the real posting being measured. In any non-clean or partially-clean workspace, TTFV-1 can pass from an older scored job and TTFV-2 can pass from an older tailored PDF even if the supplied `--job-url` never scored or never produced reviewable materials. The record stores hashes after choosing those unrelated candidates, but it never stores or checks an expected hash for the measured job.

Required fix: Derive a private expected job identity for the run (for example a hash of the submitted/canonical job key), use it to filter the API checks and navigate/select the matching UI row/item (including `/apply-review?jobKey=...` for TTFV-2), and fail when the expected job does not produce the scored job and PDF. Keep only hashes in records so the real URL remains private.

File: scripts/ttfv-real.mjs [hunk 4]
```diff
@@ -0,0 +1,676 @@
...
+}
+
+function environmentSnapshot() {
+  const cpu = os.cpus()[0];
+  return {
+    platform: os.platform(),
+    release: os.release(),
+    arch: os.arch(),
+    cpuCount: os.cpus().length,
+    cpuModel: cpu?.model ?? null,
```
Comment (ebarti) [inline scripts/ttfv-real.mjs]: PR Reviewer - Codex

Severity: Medium
Gate: Pass
Evidence: scripts/ttfv-real.mjs:602-611 stores platform release, CPU count/model, total memory, and Node version in each measurement record; docs/developer/first-run-ttfv.md:30-32 says records store hashes, counts, timings, status codes, content type, and byte length, and the record policy at scripts/ttfv-real.mjs:561 says it omits sensitive local data.
Reproduction: Reasoned from code

The measurement record captures host fingerprinting details that the privacy docs do not disclose. That may be acceptable for owner-local records, but the protocol also asks the owner to copy evidence into the packaging decision, so the record schema and docs should not understate what the JSON contains.

Required fix: Either remove or coarse-grain the hardware/environment snapshot from measurement records, or explicitly document that these fields are present and keep the generated summary/ADR guidance limited to non-identifying platform evidence.

File: scripts/ttfv-real.mjs [hunk 5]
```diff
@@ -0,0 +1,1066 @@
...
+const SCHEMA_VERSION = 1;
+const DEFAULT_API_BASE_URL = "http://127.0.0.1:8766";
+const DEFAULT_WEB_BASE_URL = "http://127.0.0.1:5173";
+const DEFAULT_TTFV_1_THRESHOLD_MS = 10 * 60 * 1000;
+const DEFAULT_TTFV_2_THRESHOLD_MS = 30 * 60 * 1000;
+const DEFAULT_WORST_MULTIPLIER = 1.5;
+const DEFAULT_TIMEOUT_MS = DEFAULT_TTFV_2_THRESHOLD_MS * DEFAULT_WORST_MULTIPLIER;
+const DEFAULT_POLL_MS = 5_000;
+const DEFAULT_INSTALL_COMMAND = "corepack pnpm install:interactive";
+const DEFAULT_INIT_COMMAND = "uv --project workers/automation run jobhunter init";
```
Comment (ebarti) [inline scripts/ttfv-real.mjs]: PR Reviewer - Codex

Severity: High
Gate: Fail
Evidence: `scripts/ttfv-real.mjs:19` and `scripts/ttfv-real.mjs:24` default the owner-run wrapper to `uv --project workers/automation run jobhunter ...`; `docs/developer/first-run-ttfv.md:64-79` tells the owner to clone `JobHunter`, `cd JobHunter`, and write records under `$HOME/.jobhunter`; current `origin/main` has already renamed the package/CLI/data-dir surface to `jobctl`/`JobCtl`/`~/.jobctl` (`package.json:2`, `workers/automation/pyproject.toml:2,62,74`, `README.md:146-151,217-225`). Live PR metadata also reports `mergeable=false` / `mergeable_state=dirty`, and `git merge-tree origin/main origin/feat/r2-real-ttfv` conflicts in `package.json`, the TTFV plan, and requirement-led tailoring tests.
Reproduction: Reasoned from code and live PR metadata

The JobCtl rename alignment is incomplete. As written, the new gateable TTFV wrapper will invoke a `jobhunter` console script that current `main` no longer exposes, and the newly added owner protocol points users at the old repo name and old app data directory. This is not just copy drift: rebasing this PR as-is would either fail the owner-run measurement path or reintroduce pre-rename package/script names while resolving the conflicts.

Required fix: Rebase onto current `origin/main`, port the TTFV wrapper/tests/docs to the live `jobctl`/`JobCtl`/`~/.jobctl` names and `@jobctl/*` package filters, resolve the merge conflicts, and rerun the no-spend script/docs/check verification on the rebased head.

**Review by ebarti — COMMENTED** [review]
Inline review finding from Codex. I could not submit this as REQUEST_CHANGES because GitHub rejects requested-changes reviews on the authenticated account's own PR.
